apps/lib/signer/signer.go

```diff
@@ -20,6 +22,19 @@ import (
 	"github.com/rs/zerolog"
 )
 
+// gmorkworld in ascii
+const StorkMagicNumber = "103109111114107119111114108100"
+const StorkAuthAssetId = "STORKAUTH"
+const StarkEncodedStorkAuthAssetId = "0x53544f524b41555448000000000000007361757468"
```

Comment:
hardcode the asset id computed from STORKAUTH with our hardcoded oracle id `sauth` so we don't need to recompute it every time.